docker/sentinel-compose.yml

```
@@ -0,0 +1,105 @@
+      echo "loadmodule /FalkorDB/bin/src/falkordb.so" >> /data/node.conf
+      redis-server /data/node.conf
+  redis-server-2:
+    image: muhammadqadora/falkor-redis
+    hostname: redis-server-2
+    container_name: redis-server-2
+    ports:
+      - 6381:6381
+    networks:
+      - redis-sentinel
+    command:
+    - bash
```

yamllint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)